requirements/test-ci-default.txt

```diff
@@ -21,4 +21,4 @@
 git+https://github.com/celery/kombu.git
 
 # SQS dependencies other than boto
-pycurl==7.43.0.5
+pycurl==7.45.2
```

Comment:
this version do not have wheels

Reply:
Yes, I probably need to add libcurl to the system dependencies.
Anyway, I was trying to run Python 3.11 unit tests locally following the contributing section of the documentation, but I'm facing import failures, so I have to dig deeper to see what's wrong

Reply:
you can check this https://github.com/celery/celery/pull/7069#issuecomment-965112783